plugins/serializers/prometheus/collection_test.go

```diff
@@ -124,39 +143,47 @@ func TestCollectionExpire(t *testing.T) {
 			name: "expired single metric in metric family",
 			now:  time.Unix(20, 0),
 			age:  10 * time.Second,
-			metrics: []telegraf.Metric{
-				testutil.MustMetric(
-					"cpu",
-					map[string]string{},
-					map[string]interface{}{
-						"time_idle": 42.0,
-					},
-					time.Unix(0, 0),
-				),
+			input: []Input{
+				{
+					metric: testutil.MustMetric(
+						"cpu",
+						map[string]string{},
+						map[string]interface{}{
+							"time_idle": 42.0,
+						},
+						time.Unix(0, 0),
+					),
+					addtime: time.Unix(0, 0),
+				},
 			},
 			expected: []*dto.MetricFamily{},
 		},
 		{
 			name: "expired one metric in metric family",
 			now:  time.Unix(20, 0),
 			age:  10 * time.Second,
-			metrics: []telegraf.Metric{
-				testutil.MustMetric(
-					"cpu",
-					map[string]string{},
-					map[string]interface{}{
-						"time_idle": 42.0,
-					},
-					time.Unix(0, 0),
-				),
-				testutil.MustMetric(
-					"cpu",
-					map[string]string{},
-					map[string]interface{}{
-						"time_guest": 42.0,
-					},
-					time.Unix(15, 0),
-				),
+			input: []Input{
+				{
+					metric: testutil.MustMetric(
+						"cpu",
+						map[string]string{},
+						map[string]interface{}{
+							"time_idle": 42.0,
+						},
+						time.Unix(0, 0),
```

Comment:
The test probably ought to have different time and addtime to show that addtime is what's used during expiration

Reply:
Added